### ISSUE TITLE
Further testing

### ISSUE THREAD

**Issue by Nyveon** [Nyveon/MCStructureCleaner#25]
Currently, only 1.15 files are tested. With the introduction of 1.18+ support in version 1.7 (#24), it would be nice to verify that everything works as expected for other versions

**Comment by ghost** [comment]
By testing do you mean users? If so, I just ran this on a 1.19.2 world that was originally 1.18, and has had many mods added and removed over the years. The tool removed 1438 instances of 6 tags. No more messages in the logs, and nothing appears to be broken.

**Comment by Nyveon** [comment]
Yeah! Manual testing helps a lot, glad to hear it worked for your case :D

There's also automated testing that can be done, but it's a bit more complicated to set up.
